database/tables/pl_sessions.pg

```diff
@@ -0,0 +1,7 @@
+columns
```

Comment:
I'd be inclined to add an indexed `user_id` column to allow ourselves to easily implement things like "destroy all sessions for user X" or "show all sessions for user Y".

Reply:
We don't get user data from the session API. But, if we know the structure of how information is stored in the JSONB blob in the database, we can good queries off of that. We can add more columns if we need it, but I'm not sure would.

Reply:
We can most likely set `req.session.user_id` in our login code and teach our store to pull that out and write it to a separate column. I agree that it's likely that performance will be good enough (and that features that require a full table scan are likely to be used infrequently), but I'd still be inclined to add this from the start, or at least in a quick followup PR.

Reply:
I didn't want to make columns we didn't use, so maybe we wait and make that a follow up PR if a JSONB query isn't good enough and we have a structured way of pulling the `user_id` out of the session data before the upsert?

Reply:
What if we add an [index](https://www.postgresql.org/docs/current/datatype-json.html#JSON-INDEXING)?

> GIN indexes can be used to efficiently search for keys or key/value pairs occurring within a large number of jsonb documents (datums).

Reply:
Hmm, I don't think we've used GIN indexes before. I'm happy to try this, but I'd also be fine to forgo an index for now until we have a use case.